src/App.tsx

```diff
@@ -1,10 +1,54 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Content } from './components/Content/Content';
+import { Notification } from './components/Notification/Notification';
 
-const USER_ID = 0;
+const USER_ID = 10888;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isAnyError, setIsAnyError] = useState(false);
+  const [isErrorMessage, setIsErrorMessage] = useState(false);
+  const [filterCondition, setFilterCondition] = useState('all');
+
+  useEffect(() => {
+    const loadTodosByUser = async (userID: number) => {
+      try {
+        const uploadedTodos = await getTodos(userID);
+
+        setTodos(uploadedTodos);
+      } catch (error) {
+        setIsAnyError(true);
+        setIsErrorMessage(true);
+        throw new Error('erroooooor');
+      }
+    };
+
+    loadTodosByUser(USER_ID);
```

Comment:
I think it's unnecessary function, you can use just function `getTodos` and simple `then catch`